tests/parser/tuple1.py

```diff
@@ -35,3 +35,5 @@ def test():
     x,
     y,
 )
+
+t : tuple[i32, i32] = 2,3    # Without bracket annotated assignment of tuples
```

Comment:
Can you please add a new line at the end of the file?

Reply:
Added but last newline is not visible from github.